### PR TITLE
Fix link to add a project to the Registry

### PR DESCRIPTION
**Pull request by austinlparker**

### CODE REVIEW

**Comment by austinlparker** [comment]
cc @mtwo if I could get a sign-off on this today it'd be good, fixes a broken link

**Comment by austinlparker** [comment]
this also fixes the issue where only one result would get returned from a search

File: layouts/registry/list.html [hunk 1]
```diff
@@ -11,7 +11,7 @@ <h1 class="title">OpenTelemetry Registry</h1>
       <p>This section of the site will allow you to discover tracers, instrumentation libraries, interfaces,
```
Comment (bhs) [inline layouts/registry/list.html]: Misc copy suggestion:

`The OpenTelemetry Registry allows you to search for instrumentation libraries, tracer implementations, utilities, and other useful projects in the OpenTelemetry ecosystem.`

Comment (austinlparker) [reply layouts/registry/list.html]: I like it!

**Pull request merged**